src/applications/vaos/services/location/index.js

```diff
@@ -239,11 +239,23 @@ export function getTestFacilityId(facilityId) {
  * @param {*} facility - facility details object
  */
 export function formatFacilityAddress(facility) {
+  if (!facility) return '';
   return `${facility.address?.line.join(', ')}, ${facility.address?.city}, ${
     facility.address?.state
   } ${facility.address?.postalCode}`;
 }
 
+/**
+ * Returns formatted facility phone number.
+ *
+ * @export
+ * @param {*} facility - facility details object
+ * @returns Formatted facility phone number.
+ */
+export function formatFacilityPhone(facility) {
```

Comment:
This doesn't appear to be doing any formatting, should we change the name of the function?

Reply:
Done.